cime_config/config_grids.xml

```diff
@@ -2755,8 +2765,6 @@
     <domain name="r05">
       <nx>720</nx>
       <ny>360</ny>
-      <file grid="atm" mask="oQU480">$DIN_LOC_ROOT/share/domains/domain.lnd.r05_oQU480.200527.nc</file>
-      <file grid="lnd" mask="oQU480">$DIN_LOC_ROOT/share/domains/domain.lnd.r05_oQU480.200527.nc</file>
```

Comment:
Why are these lines being removed?

Reply:
@iulian787 You removed these lines in https://github.com/E3SM-Project/E3SM/commit/703e752bd3a40817993f7553dcec18cfdf2e48fe